tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -131,7 +142,8 @@ def update_connection(self, workbook_item, connection_item):
     @api(version="2.0")
     @parameter_added_in(no_extract='2.5')
     @parameter_added_in(include_extract='2.5')
-    def download(self, workbook_id, filepath=None, include_extract=True, no_extract=None):
+    def download(self, workbook_id: str, filepath: Optional[Union['os.PathLike[Any]', IO[Any]]] = None,
```

Comment:
neat lines.

The filepath should be the same str-or-file as earlier, right?

Reply:
Cleaned up the lines for download. We can absolutely move the filepath type hint to its own object and reference that. I am just having difficulty getting one that plays nice with publish as well. It may end up requiring a refactoring of the publish methods.